keps/prod-readiness/sig-storage/661.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 661
+alpha:
+  approver: "@wojtek-t"
```

Comment:
Can we populate this field properly? I think both sig-storage and sig-apps should be participating/owning sigs. Also we will need sig-app approvers.

Reply:
will do

Reply:
This is purely for prod-readiness and only prod-readiness approvers are here.

SIG approvers are in kep.yaml